cves/CVE-2018-13302.yml

```diff
@@ -306,8 +320,10 @@ lessons:
     applies:
     note:
   distrust_input:
-    applies:
-    note:
+    applies: true
+    note: |
+      The substream id was assumed to be correct for all but one frame types, 
+      should have distrusted all ids
```

Comment:
```suggestion
      Instead, substream ids should have never been assumed to be correct, regardless of frame type
```

Reply:
Much better wording, thank you! Will add to final pulll request.